Data.Rmd

```diff
@@ -11,6 +11,7 @@ library(tidyverse)
 library(readstata13)
 library(skimr)
 library(psych)
+library(naniar) # use install.packages('naniar') in the console
```

Comment:
@sfergusond what does this library do?

Reply:
I used it for the old recoding method, not necessary anymore

Reply:
OK, will reject this line.